### PR TITLE
Use reanimated for the Fade component

### PR DESCRIPTION
**Pull request by dbo1dyrev**
https://user-images.githubusercontent.com/68349436/118699591-60f10200-b81a-11eb-910e-c5a14374f567.mov

### CODE REVIEW

**Review by thehobbit85 — CHANGES_REQUESTED** [review]
See comments

File: src/components/themed/Fade.js [hunk 1]
```diff
@@ -1,86 +1,54 @@
 // @flow
 
 import * as React from 'react'
-import { type AnimatedValue, Animated } from 'react-native'
+import Animated, { interpolate, useAnimatedStyle, useSharedValue, withTiming } from 'react-native-reanimated'
 
-type OwnProps = {
-  timing?: 'ease' | 'linear',
+// $FlowFixMe
```
Comment (thehobbit85) [inline src/components/themed/Fade.js:6]: The whole point of having the "future-commit" was to remove this `$FlowFixMe` ;)

**Review by thehobbit85 — CHANGES_REQUESTED** [review]
Forgot to remove the `$FlowFixMe` ;)

**Comment by dbo1dyrev** [comment]
Start using `util/hooks`

**Review by thehobbit85 — APPROVED** [review]

**Pull request merged**